security-force-monitor/sfm-cms
# Revert PR adding download redirect

## Overview

This PR re-enables downloads by reverting #723.

Connects #722.

## Testing Instructions

* Run the site as described in the README and confirm you can access the downloads page.
